beta-20160602.md

```diff
@@ -0,0 +1,29 @@
+---
+title: What's New in beta-20160602
+toc: false
+---
+
+## Jun 2, 2016
+
+### New Features
+
+* String literals can now be parsed as `DATE`, `TIMESTAMP`, `TIMESTAMPTZ` or `INTERVAL` without an explicit cast. [#6925](https://github.com/cockroachdb/cockroach/pull/6925)
+* Floor division is now supported with a new operator `//`. [#6642](https://github.com/cockroachdb/cockroach/pull/6642)
+* Sub-queries are now allowed in `LIMIT`, `OFFSET`, and `RETURNING` expressions. [#6735](https://github.com/cockroachdb/cockroach/pull/6735)
+
+### Bug Fixes
+
+* Fixed a missing error check that could result in inconsistences when transactions conflict. [#6899](https://github.com/cockroachdb/cockroach/pull/6899)
+* Indexes and tables that are referenced by foreign keys may not be dropped. [#6832](https://github.com/cockroachdb/cockroach/pull/6832)
```

Comment:
You can create foreign keys now, but the actual effects are going in piecemeal (kind of like CHECK constraints). I think this is the first time foreign keys have had any effect, so we didn't mention them in the release notes when they were added. Given that, we should probably leave them out of the docs/notes until they're more completely implemented. @dt, can you confirm?